src/webgl-preview.ts

```diff
@@ -310,17 +323,54 @@ export class WebGLPreview {
     }
 
     if (this.renderTubes) {
-      console.warn('Volumetric rendering is experimental and may not work as expected or change in the future.');
+      console.warn('Volumetric rendering is experimental. It may not work as expected or change in the future.');
       const light = new AmbientLight(0xcccccc, 0.3 * Math.PI);
       // threejs assumes meters but we use mm. So we need to scale the decay of the light
       const dLight = new PointLight(0xffffff, Math.PI, undefined, 1 / 1000);
       dLight.position.set(0, 500, 500);
       this.scene.add(light);
       this.scene.add(dLight);
     }
+  }
+
+  createGroup(name: string): Group {
```

Comment:
Is this public? I think manipulating groups directly from outside the library can have unintended consequences, but can also be useful to add things to the scene. It should probably remain private until we get more confidence.